rollup.config.js

```diff
@@ -12,15 +12,24 @@ export default {
     esbuild({
       target: "es6",
     }),
+    [
```

Comment:
nothing changed about the rails dependency, so there shouldn't be changes in this file

Reply:
fixed